packages/amplify_api_plugin_interface/lib/src/GraphQL/GraphQLRequest.dart

```diff
@@ -13,16 +13,14 @@
  * permissions and limitations under the License.
  */
 
-import 'package:flutter/foundation.dart';
-
 import '../UUID.dart';
 
 class GraphQLRequest<T> {
```

Comment:
I'm not 100% sure either - Ashish can give a definitive answer. 

From what I understand, adding the type parameter there allows other methods to know what type to use as well.  

amplify_api.dart: 
`  @override
  GraphQLOperation<T> query<T>({@required GraphQLRequest<T> request}) {
    return _instance.query(request: request);
  }`

But when we call this query method, we just need to provide a GraphQLRequest<String> instead of calling query<String> as shown in the documentation: 

https://docs.amplify.aws/lib/graphqlapi/query-data/q/platform/flutter
```

 var operation = Amplify.API.query(
        request: GraphQLRequest<String>(
```